fully_documented.txt

```diff
@@ -10,6 +10,7 @@ backpack/core/derivatives/permute.py
 backpack/core/derivatives/lstm.py
 backpack/core/derivatives/linear.py
 backpack/core/derivatives/adaptive_avg_pool_nd.py
+backpack/core/derivatives/batchnorm_nd.py
 
```

Comment:
Add BN-related files from tests, too.

Reply:
Honestly, there was not much to add.
I additionally added the bn-settings.
I'm leaving out all the bn-extensions because they will be featured in the next pull request.

Reply:
Close.